src/flask/schemas/chef_schema.py

```diff
@@ -0,0 +1,21 @@
+# schemas fro serializing entities
+from marshmallow import Schema, fields, post_load
+from src.lib.entities.sqlalchemy_orm_mapping import Chef
+
+
+class ChefSchema(Schema):
+
+    user_id = fields.Int()
+    skill = fields.Int()
+    created_by = fields.Int()
+    updated_by = fields.Int()
+
+    @post_load
+    def create_sqlalchemy_chef_entity(self, data, **kwargs):
+        chef = Chef()
```

Comment:
What about creating the general utility for this?

Reply:
done.